packages/components/src/components/multi-select/multi-select/multi-select-utils.ts

```diff
@@ -0,0 +1,166 @@
+import { FormState } from '../../../utils/form/form-state';
+import {
+  determineDropdownDirection,
+  SelectDropdownDirection,
+  SelectDropdownDirectionInternal,
+} from '../../../utils/select/select-dropdown';
+import { Theme } from '../../../utils';
+import { forceUpdate } from '@stencil/core';
+import { MultiSelectOptionInternalHTMLProps } from '../multi-select-option/multi-select-option-utils';
+
+export type MultiSelectState = FormState;
+export type MultiSelectDropdownDirection = SelectDropdownDirection;
+
+export const syncMultiSelectOptionProps = (options: HTMLPMultiSelectOptionElement[], theme: Theme): void => {
+  options.forEach((item: HTMLElement & MultiSelectOptionInternalHTMLProps) => {
+    item.theme = theme;
+    forceUpdate(item);
```

Comment:
Is `forceUpdate` really needed since we are currently about to render the parent. Or are children already done rendering and we rerender them again?